recon/src/libp2p.rs

```diff
@@ -229,33 +249,34 @@ impl<R: Recon> NetworkBehaviour for Behaviour<R> {
     ) {
         match event {
             // The peer has started to synchronize with us.
-            FromHandler::Started => self.peers.entry(peer_id).and_modify(|info| {
-                if info.status != PeerStatus::Started {
-                    info.status = PeerStatus::Started;
-                    self.swarm_events_queue.push_front(Event::PeerEvent {
+            FromHandler::Started { stream_set } => self.peers.entry(peer_id).and_modify(|info| {
+                info.status = PeerStatus::Started { stream_set };
```

Comment:
Originally there was a check here that we weren't already started, to prevent a double publish. Should that check be re-added? What is the expected behavior if the peer calls started twice, especially with different stream sets?